config/account.go

```diff
@@ -335,6 +335,7 @@ func (m AccountModules) ModuleConfig(id string) (json.RawMessage, error) {
 
 type AccountPrivacy struct {
 	AllowActivities *AllowActivities `mapstructure:"allowactivities" json:"allowactivities"`
+	DSA             *AccountDSA      `mapstructure:"dsa" json:"dsa"`
```

Comment:
We need to add support for DSA in SetupViper for the default account.

Reply:
I think we are already covered here. The host can specify the default account DSA via `account_defaults.privacy.dsa` and it should default to nil so I didn't see a need to explicitly set the default. Am I missing something?

Reply:
We need to register the environment variables with Viper, otherwise the fields will be ignored. You don't need to use `v.SetDefault` since we're covered on the actual default value, but we need to at least call `v.Bind` to register the variable. `v.SetDefault` includes a call to `v.Bind` and sometimes it just looks nicer. Whichever approach you want to use is fine.

Reply:
👍 I keep forgetting about that case!